post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-configuration.adoc

```diff
@@ -22,21 +22,75 @@ For information on migrating your single-architecture cluster to a cluster that
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-aws.adoc#creating-multi-arch-compute-nodes-aws[Creating a cluster with multi-architecture compute machines on AWS]
+|Amazon Web Services (AWS)
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc#creating-multi-arch-compute-nodes-bare-metal[Creating a cluster with multi-architecture compute machines on bare metal]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.adoc#creating-multi-arch-compute-nodes-gcp[Creating a cluster with multi-architecture compute machines on GCP]
+|Google Cloud Platform (GCP)
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc#creating-multi-arch-compute-nodes-ibm-z[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with z/VM]
+.3+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc#creating-multi-arch-compute-nodes-bare-metal[Creating a cluster with multi-architecture compute machines on bare metal, {ibm-power-title}, or {ibm-z-title}]
+|Bare metal
+|&#10003;
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm.adoc#creating-multi-arch-compute-nodes-ibm-z-kvm[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with {op-system-base} KVM]
+|{ibm-power-title}
+|&#10003;
+|
+|`x86_64` or `ppc64le`
+|`x86_64`, `ppc64le`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc#creating-multi-arch-compute-nodes-ibm-power[Creating a cluster with multi-architecture compute machines on {ibm-power-name}]
+|{ibm-z-title}
+|&#10003;
+|
+|`x86_64` or `s390x`
+|`x86_64`, `s390x`
+
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc#creating-multi-arch-compute-nodes-ibm-z[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with z/VM]
+|{ibm-z-name} and {ibm-linuxone-name}
+|&#10003;
+|
+|`x86_64`
+|`x86_64`, `s390x`
+
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm.adoc#creating-multi-arch-compute-nodes-ibm-z-kvm[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with {op-system-base} KVM]
+|{ibm-z-name} and {ibm-linuxone-name}
+|&#10003;
```

Comment:
are these installations on linux one supported configurations? have we checked with Duncan and others?

Reply:
I believe so? @SNiemann15 @prb112

Reply:
I can't speak to LInuxOne.

Reply:
Should I remove it from the platforms part?

Reply:
Please keep LinuxONE it is supported.  
fyi @holgwolf